docker/docker-compose.consumers-without-neo4j.yml

```diff
@@ -27,5 +27,3 @@ services:
     container_name: datahub-mce-consumer
     ports:
       - "9090:9090"
-    depends_on:
```

Comment:
We initially architected with datahub-gms serving as a front for elastic and MySQL, hiding the consumers from the complexity of knowing how to talk to each state store and encapsulating the complex configurations.. Really want to be sure this is worth splitting open

Reply:
The concept is preserved in the sense that a common library still abstracts the storage layers. The exact same library components are used, we are merely changing the routing of the requests at runtime to bypass the GMS bottleneck. The datastore configuration parameters are shared in the helm chart so no duplication there. I believe the complexity of the configuration is no greater for the end user than it is today.